### PR TITLE
Add mergify

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: .github/mergify.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        method: squash
+        commit_message: title+body
+        bot_account: jaegerci-bot
+  - name: Kafka changes
+    conditions:
+    - files~=kafka
+    actions:
+      assign:
+        users:
+          - rubenvp8510
```
Comment (jpkrohling) [inline .github/mergify.yml:31]: Are you ok in owning everything related to Kafka?

File: .github/mergify.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          - jpkrohling
+  - name: Upgrade mechanism changes
+    conditions:
+    - files~=upgrade/
+    actions:
+      assign:
+        users:
+          - rubenvp8510
```
Comment (jpkrohling) [inline .github/mergify.yml:42]: And upgrade?

File: .github/mergify.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+          - jpkrohling
+  - name: Sidecar injection mechanism changes
+    conditions:
+    - files~=inject/
+    actions:
+      assign:
+        users:
+          - rubenvp8510
```
Comment (jpkrohling) [inline .github/mergify.yml:53]: And sidecar injection?

File: .github/mergify.yml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+          - jpkrohling
+  - name: Streaming changes
+    conditions:
+    - files~=streaming
+    actions:
+      assign:
+        users:
+          - objectiser
```
Comment (jpkrohling) [inline .github/mergify.yml:64]: Same question to you, @objectiser :-)

File: .github/mergify.yml (hunk 1 above)
Comment (rubenvp8510) [reply .github/mergify.yml:31]: +1

File: .github/mergify.yml (hunk 2 above)
Comment (rubenvp8510) [reply .github/mergify.yml:42]: +1

File: .github/mergify.yml (hunk 3 above)
Comment (rubenvp8510) [reply .github/mergify.yml:53]: Same here :) +1

File: .github/mergify.yml (hunk 4 above)
Comment (objectiser) [reply .github/mergify.yml:64]: ok

**Review by objectiser — APPROVED** [review]

**Pull request merged**